src/main/kotlin/uk/gov/justice/digital/assessments/config/CacheConfiguration.kt

```diff
@@ -33,25 +32,28 @@ class CacheConfiguration {
   @Value("\${cache.ttlMinutes.default}")
   var defaultCacheTtlMinutes: Long = 5
 
-  @Bean
-  fun defaultRedisCacheConfiguration(): RedisCacheConfiguration {
-    return getDefaultCacheConfiguration()
-      .entryTtl(Duration.ofMinutes(defaultCacheTtlMinutes))
+  private fun objectMapper(): ObjectMapper {
+    return ObjectMapper()
```

Comment:
seems very odd to register your own object mapper instead of using the one supplied with spring